Packs/CrowdStrikeHost/Integrations/integration-CrowdStrikeHost.yml

```diff
@@ -483,6 +483,74 @@ script:
         };
     }
 
+    function threatGraphSummary() {
+        var ids = argToList(args.ctg_id);
+        var res = doReq('GET', 'threatgraph/combined/control-graphs/summary/v1', {ids: ids});
+        var md = '## Falcon Host Threat Graph Summary\n';
+        var ec = {};
+        if (res.obj.resources && res.obj.resources.length > 0) {
+            ec= {"CrowdStrikeHost.ThreatGraphSummary": res.obj.resources};
```

Comment:
no merging DT. maybe there is an ID we can use to merge? see other commands implementation.